contrib/config_regtest.toml

```diff
@@ -0,0 +1,8 @@
+daemon = false
+log_level = "trace"
+data_dir = "/path/to/your/data_dir"
+listen = "0.0.0.0:8383"
```

Comment:
Hmm, it's a tough one: they don't need to listen on all interface, only localhost in your revaultd tutorial. It's nice to have this default for usability, but not security :shrug: 

Anyways, you'll need to modify the port or they will all listen on the same one (and the coordinator too)

Reply:
Updated, not sure if it makes sense to say "# If the cosigner and the managers are not in the same network, you'll need to update this with a less strict rule"